ProjectObsidian.SourceGenerators/BindingGenerator.cs

```diff
@@ -150,7 +152,7 @@ namespace {BindingPrefix}{_currentNameSpace};
 {_genericTypesAttribute}
 {_oldTypeNameAttribute}
 [Category(new string[] {{""ProtoFlux/Runtimes/Execution/Nodes/{_category}""}})]
-public partial class {_fullName} : global::FrooxEngine.ProtoFlux.Runtimes.Execution.{_baseType} {_constraints}
```

Comment:
Does this break functionality of other nodes?

Reply:
I don't think so